cassandraprotocol/constants.go

```diff
@@ -185,8 +190,8 @@ const (
 	HeaderFlagUseBeta       = HeaderFlag(0x10)
 )
 
-// query flags were [byte] in v3 and v4, but changed to [int] in v5
-type QueryFlag = int32
+// Note: QueryFlag was encoded as [byte] in v3 and v4, but changed to [int] in v5
+type QueryFlag = uint32
```

Comment:
I changed all the flags to uint32 because DSE really utilizes the entire uint32 range.